iocore/net/OCSPStapling.cc

```diff
@@ -46,13 +40,217 @@
 
 extern ClassAllocator<FetchSM> FetchSMAllocator;
 
```

Comment:
Why can' t the anonymous namespace start here?

Reply:
I don't remember the exact error, but it was probably considered as a different thing if it's in a namespace because the actual FetchSMAllocator is not in any namespace.

Reply:
I meant the anonymous namespace should start after that declaration, not include it.

Reply:
```
  CXX      OCSPStapling.o
OCSPStapling.cc:230:1: error: no template named 'StackTraits'; did you mean '::bssl::internal::StackTraits'?
DEFINE_STACK_OF(TS_OCSP_ONEREQ)
^
/Users/mkitajo/opt/boringssl/include/openssl/stack.h:100:31: note: expanded from macro 'DEFINE_STACK_OF'
#define DEFINE_STACK_OF(type) DEFINE_NAMED_STACK_OF(type, type)
                              ^
/Users/mkitajo/opt/boringssl/include/openssl/stack.h:94:3: note: expanded from macro 'DEFINE_NAMED_STACK_OF'
  BORINGSSL_DEFINE_STACK_TRAITS(name, type, false)
  ^
/Users/mkitajo/opt/boringssl/include/openssl/stack.h:357:10: note: expanded from macro 'BORINGSSL_DEFINE_STACK_TRAITS'
  struct StackTraits<STACK_OF(name)> {                      \
         ^
/Users/mkitajo/opt/boringssl/include/openssl/stack.h:347:8: note: '::bssl::internal::StackTraits' declared here
struct StackTraits {};
       ^
OCSPStapling.cc:230:1: error: class template specialization of 'StackTraits' not in a namespace enclosing 'internal'
DEFINE_STACK_OF(TS_OCSP_ONEREQ)
^
/Users/mkitajo/opt/boringssl/include/openssl/stack.h:100:31: note: expanded from macro 'DEFINE_STACK_OF'
#define DEFINE_STACK_OF(type) DEFINE_NAMED_STACK_OF(type, type)
                              ^
/Users/mkitajo/opt/boringssl/include/openssl/stack.h:94:3: note: expanded from macro 'DEFINE_NAMED_STACK_OF'
  BORINGSSL_DEFINE_STACK_TRAITS(name, type, false)
  ^
/Users/mkitajo/opt/boringssl/include/openssl/stack.h:357:10: note: expanded from macro 'BORINGSSL_DEFINE_STACK_TRAITS'
  struct StackTraits<STACK_OF(name)> {                      \
         ^
blah blah blah
```